pages/_app.js

```diff
@@ -78,7 +79,9 @@ export default class Root extends App {
         onRedirectCallback={onRedirectCallback}
       >
         <Layout>
-          <Component {...pageProps} />
+          <AppData>
```

Comment:
Placing `<AppData>` here is not playing well with `<NavBar>`. (And placing `<AppData>` as parent of `<Layout>` doesn't work because the components don't get the user props.) I can move the logged-in user fetching code back to inside `<MyLayout>` to make this work again. Thoughts?